extras/drift_postgres/lib/src/pg_database.dart

```diff
@@ -0,0 +1,202 @@
+import 'package:drift/backends.dart';
+import 'package:postgres/postgres.dart';
+
+///
+class PgDatabase extends DelegatedDatabase {
+  ///
+  PgDatabase(PostgreSQLConnection connection)
+      : super(_PgDelegate(connection, connection),
+            isSequential: true, logStatements: true);
+
+  ///
+  factory PgDatabase.open(String host, int port, String databaseName,
```

Comment:
I'm not sure if we need this constructor. We'll just always have to catch up with the API from `package:postgres` and the convenience for users appears to be small.

Reply:
It is a method for connect without specify postgres dependency in pubspec. But, I can remove if you think it isn't good

Reply:
Yeah I think I would prefer users to import `postgres` too, they are implicitly using its API.